packages/react-core/src/helpers/util.ts

```diff
@@ -314,15 +315,49 @@ export const formatBreakpointMods = (
     }
     return '';
   }
+
   return Object.entries(mods || {})
-    .map(([breakpoint, mod]) => `${stylePrefix}${mod}${breakpoint !== 'default' ? `-on-${breakpoint}` : ''}`)
+    .map(
+      ([breakpoint, mod]) =>
+        `${stylePrefix}${mod}${breakpoint !== 'default' ? `-on-${breakpoint}` : ''}${
+          vertical && breakpoint !== 'default' ? '-height' : ''
+        }`
+    )
     .map(toCamel)
     .map(mod => mod.replace(/-?(\dxl)/gi, (_res, group) => `_${group}`))
     .map(modifierKey => styles.modifiers[modifierKey])
     .filter(Boolean)
     .join(' ');
 };
 
+/**
+ * Return the breakpoint for the given height
+ *
+ * @param {number | null} height The height to check
+ * @returns {'default' | 'sm' | 'md' | 'lg' | 'xl' | '2xl'} The breakpoint
+ */
+export const getVerticalBreakpoint = (height: number): 'default' | 'sm' | 'md' | 'lg' | 'xl' | '2xl' => {
+  if (height === null) {
+    return null;
+  }
+  if (height >= 1280) {
```

Comment:
Just curious if these could be brought in another way so they aren't hardcoded. They should be available as react tokens if that can be used since they're CSS vars - https://github.com/patternfly/patternfly/blob/5a2e36f9f6db8323951a60438fe0cd1b7157a628/src/patternfly/base/_variables.scss#L201-L206

I guess you would just want to strip the unit?